### PR TITLE
Added browser support info

### PR DESCRIPTION
**Pull request by mr-mig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,10 +23,30 @@ Bear in mind:
 
 Table of contents:
 
+* [Browsers Support](#browsers-support)
 * [Methodology](#methodology)
 * [How To](#howto)
 * [Contribution](#contribution)
 
+## Browsers Support
```
Comment (kdzwinel) [inline README.md]: "Browser support" or "Supported browsers"

File: README.md [hunk 2]
```diff
@@ -23,10 +23,30 @@ Bear in mind:
...
+
+We support the following browsers:
+
+| Browser | Versions  | Total share in global traffic |
+| ---- | ---- | ---- |
+| Google Chrome | 42+ | 44.69% |
+| Safary | 8+ | 10.22% |
```
Comment (kdzwinel) [inline README.md]: Safari

**Comment by vergilius** [comment]
- how often we should update this?
- how should look process in development/testing with all these browsers? Testing each browser in the list every major change?

**Comment by mr-mig** [comment]
Let's update it at least every 3 month.

Regarding testing - we need to discuss it, as well as the approaches to deal with features/refactors

**Comment by vergilius** [comment]
:+1:

**Comment by aju** [comment]
:ok:

**Pull request merged**